janusgraph-examples/example-remotegraph/README.md

```diff
@@ -0,0 +1,75 @@
+# Remote Graph
+
+## About Remote Graph
+
+[Gremlin Server](http://tinkerpop.apache.org/docs/3.2.6/reference/#gremlin-server)
+allows a JanusGraph instance to run on a standalone server. This enables
+multiple clients to connect to the same JanusGraph instance. It also allows
+clients to connect via non-JVM languages. By default, the Gremlin Server
+communicates over WebSocket.
+
+A Java program can connect to the Gremlin Server in two ways:
+* [Gremlin Driver](http://tinkerpop.apache.org/docs/3.2.6/reference/#connecting-via-java)
+client. Sumbit scripts as a `String` for evaluation on the server.
+* [Remote Graph](http://tinkerpop.apache.org/docs/3.2.6/reference/#connecting-via-remotegraph)
+uses the same client connection but enables you to compose queries as code.
```

Comment:
It might be easy to forget to update the TinkerPop link versions. Is this common throughout the docs and/or do we need an issue to track adding support for automatically updating version when we upgrade TinkerPop?

Reply:
Good call, there's a syntax to use where the value can be replaced with the Maven value. I'll update this.